swagger.yaml

```diff
@@ -1093,6 +1094,264 @@ paths:
               schema:
                 $ref: "#/components/schemas/Error"
 
+  "/card-comments":
+    post:
+      operationId: createComment
+      summary: Add a comment to a Trello card
+      description: Creates a new comment on a specified Trello card.
+      tags:
+        - Trello
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                cardId:
+                  type: string
+                  description: The ID of the Trello card.
+                comment:
+                  type: string
+                  description: The comment text.
+      responses:
+        "200":
+          description: Comment created successfully.
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  cardId:
+                    type: string
+                    description: The ID of the Trello card.
+                  comment:
+                    type: string
+                    description: The comment text.
+        "400":
+          description: Missing required parameters.
+        "500":
+          description: Failed to create comment.
+
+  "/card":
+    post:
+      operationId: createCard
+      summary: Create a new Trello card
+      description: Adds a new card to a Trello board.
+      tags:
+        - Trello
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                title:
+                  type: string
+                  description: The title of the Trello card.
+                description:
+                  type: string
+                  description: Description of the Trello card.
+      responses:
+        "200":
+          description: Card created successfully.
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  cardId:
+                    type: string
+                  title:
+                    type: string
+                  description:
+                    type: string
```

Comment:
Probably worth making a TrelloCard schema for this

Reply:
Actually it seems you already have created the schema Card. So it should be referenced here also